pkg/converter/convert_test.go

```diff
@@ -213,14 +213,30 @@ func TestConvert(t *testing.T) {
 			},
 		},
 		{
-			name:         "isExecUsingkubelogin",
-			execArgItems: []string{getTokenCommand, argEnvironment, envName, argServerID, serverID, argClientID, clientID, argTenantID, tenantID, argLoginMethod},
+			name:         "exec format kubeconfig azurecli",
+			execArgItems: []string{getTokenCommand, argEnvironment, envName, argServerID, serverID, argClientID, clientID, argTenantID, tenantID, argLoginMethod, "azurecli"},
```

Comment:
it's converting from azurecli to azurecli? i mean, it's definitely one of supported scenarios, but lets add all the combinations